setup.py

```diff
@@ -76,7 +76,7 @@ def _find_symlinks(topdir, extension=''):
                     else:
                         # If we managed to stay within the tree, store the symlink
                         link = filepath[len(topdir):]
-                        if link.startswith('/'):
+                        if link.startswith(os.path.sep):
```

Comment:
I guess this should also be `os.path.isabs(link)`, but the next line also needs to be adjusted.